plugins/parodos/src/hooks/useWorkflowDefinitionToJsonSchema/jsonSchemaFromWorkflowDefinition.ts

```diff
@@ -10,7 +10,7 @@ import { taskDisplayName } from '../../utils/string';
 import type { StrictRJSFSchema, UiSchema } from '@rjsf/utils';
 import { assert } from 'assert-ts';
 
-export function getJsonSchemaType(type: ParameterFormat) {
+export function getJsonSchemaType(type: ParameterFormat, options: string[]) {
```

Comment:
```suggestion
export function getJsonSchemaType(type: ParameterFormat, options: string[] = []) {
```

goes with suggestion on 138.  Default can do it in one line